### PR TITLE
Add example security rules for OCI VCNs

### PR DESCRIPTION
**Pull request by rhomast**
To hopefully make it clear that these are examples and shouldn't be copied wholesale, I've put the tables in sections that are collapsed by default. If you think that's too hidden, let me know.

### CODE REVIEW

File: content/en/docs/setup/install/prepare/platforms/OLCNE/vcn.md [hunk 1]
```diff
@@ -43,6 +46,27 @@ In this subnet, create security rules that cover the following traffic:
...
+
+| Destination Type | Destination | Destination Port | Protocol | Description |
+|------------------|-------------|------------------|----------|-------------|
+| CIDR Block       | 0.0.0.0/0   | 6443             | TCP      | Public access to endpoint OCI load balancer |
+| CIDR Block       | 10.0.0.0/16 |                  | ICMP Type 3, Code 4 | Path MTU discovery |
+</details>
+
 ### Subnet 2: control plane nodes
 
 A private subnet that houses the control plane nodes that run Kubernetes control plane components such as the API Server and the control plane pods.
```
Comment (rosemarymarano) [inline content/en/docs/setup/install/prepare/platforms/OLCNE/vcn.md:72]: -> control plane components, (comma) such as the API Server

File: content/en/docs/setup/install/prepare/platforms/OLCNE/vcn.md [hunk 2]
```diff
@@ -61,6 +85,38 @@ In this subnet, create security rules that cover the following traffic:
...
+
+{{< alert title="NOTE" color="primary" >}}
+These examples are provided for reference only. Customize your security rules as needed for your environment.
+{{< /alert >}}
+
+#### Egress rules
+
+| Destination Type | Destination | Destination Port | Protocol | Description |
+|------------------|-------------|------------------|----------|-------------|
+| CIDR Block       | 0.0.0.0/0   | All              | All      | Control plane node access to the internet to pull images |
```
Comment (rosemarymarano) [inline content/en/docs/setup/install/prepare/platforms/OLCNE/vcn.md:99]: -> Internet

File: content/en/docs/setup/install/prepare/platforms/OLCNE/vcn.md [hunk 3]
```diff
@@ -88,6 +165,35 @@ In this subnet, create security rules that cover the following traffic:
...
+
+{{< alert title="NOTE" color="primary" >}}
+These examples are provided for reference only. Customize your security rules as needed for your environment.
+{{< /alert >}}
+
+#### Egress rules
+
+| Destination Type | Destination | Destination Port | Protocol | Description |
+|------------------|-------------|------------------|----------|-------------|
+| CIDR Block       | 0.0.0.0/0   | All              | All      | Worker node access to the internet to pull images |
```
Comment (rosemarymarano) [inline content/en/docs/setup/install/prepare/platforms/OLCNE/vcn.md:179]: -> Internet

File: content/en/docs/setup/install/prepare/platforms/OLCNE/vcn.md [hunk 4]
```diff
@@ -88,6 +165,35 @@ In this subnet, create security rules that cover the following traffic:
...
+| CIDR Block       | 10.0.0.0/29  |                  | IP-in-IP | Calico networking with IP-in-IP enabled |
+| CIDR Block       | 10.0.64.0/20 |                  | IP-in-IP | Calico networking with IP-in-IP enabled |
+| CIDR Block       | 10.0.0.0/16  |                  | ICMP Type 3, Code 4 | Path MTU discovery |
+| CIDR Block       | 0.0.0.0/0    | 22               | 22       | Inbound SSH traffic to worker nodes |
+| CIDR Block       | 10.0.0.0/16  | All              | TCP      | East-West communication for Kubernetes API server access / DNS access |
+</details>
+
 ## Gateways
 
 Gateways control access from your VCN to other networks. You'll need to configure three different types of gateways:
```
Comment (rosemarymarano) [inline content/en/docs/setup/install/prepare/platforms/OLCNE/vcn.md:199]: -> Internet (globally)

**Review by rosemarymarano — COMMENTED** [review]
Few edits

File: content/en/docs/setup/install/prepare/platforms/OLCNE/vcn.md (hunk 4 above)
Comment (rhomast) [reply content/en/docs/setup/install/prepare/platforms/OLCNE/vcn.md:199]: The style guide actually now recommends lowercase. I've changed Internet to uppercase here to blend with its occurrences in the rest of the documentation but we might want to consider replacing all instances of 'Internet' with 'internet' throughout the docs. There don't seem to be many.

File: content/en/docs/setup/install/prepare/platforms/OLCNE/vcn.md [hunk 5]
```diff
@@ -61,6 +85,38 @@ In this subnet, create security rules that cover the following traffic:
     * BGP
     * IP-in-IP
 
+<details>
+<summary>Security rule examples</summary>
```
[the remark this answers is not in the document]
Comment (rhomast) [reply content/en/docs/setup/install/prepare/platforms/OLCNE/vcn.md:89]: The double plurals feels awkward.

File: content/en/docs/setup/install/prepare/platforms/OLCNE/vcn.md (hunk 4 above)
Comment (rosemarymarano) [reply content/en/docs/setup/install/prepare/platforms/OLCNE/vcn.md:199]: Huh, did not know that! Thanks for the heads up.

**Comment by kminder** [comment]
I still think the networking diagram is needed.

**Comment by kminder** [comment]
The tables are helpful including the alerts that some of the values are only examples.  I do think a little more details required to help a reader understand the "environment" from which the values came from.  This is somewhat related to the diagram.  In general I'm thinking of some way to link the various CIDR blocks in the examples back to a hypothetical network configuration.

**Comment by rhomast** [comment]
> 

I've sent the diagram to the graphics team and it will be added later.

**Review by theabmitra — APPROVED** [review]
Content lgtm, but the diagram is needed.

**Review by rosemarymarano — APPROVED** [review]

**Pull request merged**